seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/CoordinatorService.java

```diff
@@ -275,6 +278,16 @@ private void initCoordinatorService() {
         if (connectorJarStorageConfig.getEnable()) {
             connectorPackageService = new ConnectorPackageService(seaTunnelServer);
         }
+        // task log manager service
+        if (engineConfig.getTelemetryConfig() != null
```

Comment:
This seems to be initialized when the master starts. It looks like it's only created on the master node. Worker nodes do not seem to be considered.